src/js/Content/Features/Community/Inventory/FInventoryMarketHelper.js

```diff
@@ -13,17 +13,15 @@ export default class FInventoryMarketHelper extends Feature {
         Page.runInPageContext(() => {
 
             /* eslint-disable no-undef, camelcase */
-            window.SteamFacade.jq(document).on("click", ".inventory_item_link, .newitem", () => {
-                if (!g_ActiveInventory.selectedItem.description.market_hash_name) {
-                    g_ActiveInventory.selectedItem.description.market_hash_name = g_ActiveInventory.selectedItem.description.name;
-                }
-                let market_expired = false;
-                if (g_ActiveInventory.selectedItem.description) {
-                    market_expired = g_ActiveInventory.selectedItem.description.descriptions.reduce(
-                        (acc, el) => (acc || el.value === "This item can no longer be bought or sold on the Community Market."),
-                        false,
-                    );
-                }
+            document.addEventListener("click", ({target}) => {
```

Comment:
You sure you want the whole document to listen to this click event? Wouldn't it be better to just listen to the click events of the target elements?

Reply:
I believe it's the same as before, only in native JS methods. I don't intend to further change this logic in this PR.